lib/sycamore/sycamore/tests/unit/query/test_plan.py

```diff
@@ -159,7 +159,7 @@ def vector_search_filter_plan():
 
 
 def get_logical_plan(plan):
```

Comment:
can we add a test for post_process as well

Reply:
Added unit tests.